src/activity/events.jl

```diff
@@ -98,17 +98,17 @@ function handle_event_request(request, handle;
                               secret = nothing, events = nothing,
                               repos = nothing, forwards = nothing)
     if !(isa(secret, Void)) && !(has_valid_secret(request, secret))
-        return HTTP.Response(400, "invalid signature")
```

Comment:
Not sure how to add these messages.

Reply:
cc @quinnj @samoconnor

Reply:
@quinnj, @KristofferC, looking at this now...

Reply:
See https://github.com/JuliaWeb/HTTP.jl/commit/2c14d438cf44cceb5c6bba41c51a92f8715e543a
 -- " add constructors: Response(s::Int, body::AbstractString) ..."

Also, maybe this catch is not required because of the catch in [`HTTP.Servers.handle_stream()`](https://github.com/JuliaWeb/HTTP.jl/blob/master/src/Servers.jl#L463)
```julia
        server = HTTP.Servers.Server() do request, response
            try
                handle_event_request(request, handle; auth = auth,
                                     secret = secret, events = events,
            vvvvv                    repos = repos, forwards = forwards)
            catch err
                bt = catch_backtrace()
                print(STDERR, "SERVER ERROR: ")
                Base.showerror(STDERR, err, bt)
                return HTTP.Response(500)
            end
        end
```